lib/rubocop/cop/mixin/regexp_metacharacter.rb

```diff
@@ -21,6 +21,10 @@ def literal_at_end?(regex_str)
         regex_str =~ /\A(?:#{Util::LITERAL_REGEX})+(\\z|\$)\z/
       end
 
+      def repeating_literal?(regex_str)
+        regex_str.match?(/\A(?:#{Util::LITERAL_REGEX})\+\z/)
+      end
```

Comment:
It may be moved to `Performance/Squeeze` if this method is not used for other than `Performance/Squeeze`.